src/styles/brackets_fonts.less

```diff
@@ -30,7 +30,11 @@
 /* SourceSansPro Regular */
 @font-face {
     font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Regular.ttf');
+    src: local('Source Sans Pro Regular'),
+         url('fonts/SourceSansPro/SourceSansPro-Regular.eot?iefix') format('embedded-opentype'),
```

Comment:
I wonder if we actually need this, AFAICT this only targets IE<9, and as Brackets' in-browser variant is thought to be for modern browsers only (at least that's what I know), I don't think we need this line.

cc @humphd @Pomax